tests/templates/kuttl/spark-history-server/helm-bitnami-eventlog-minio-values.yaml.j2

```diff
@@ -0,0 +1,55 @@
+---
+mode: standalone
+disableWebUI: false
+extraEnvVars:
+  - name: BITNAMI_DEBUG
+    value: "true"
+  - name: MINIO_LOG_LEVEL
+    value: DEBUG
+
+provisioning:
+  enabled: true
+  buckets:
+    - name: spark-logs/eventlogs
```

Comment:
I'm not sure if this is a correct bucket name (because of the / )  `eventlogs` is enough

Reply:
It seems to work: I'd sort of like to leave it as it is so that we have tested a multilevel bucket name, should someone need that. What do you think?